### PR TITLE
fix(iamRole): relate iamPermissionsBoundary policy, include inlinePolicies JSON objects

### PR DESCRIPTION
**Pull request by hjaraujof**
## Issue tracker links

[CG-1262](https://autoclouddev.atlassian.net/browse/CG-1262?atlOrigin=eyJpIjoiMGVlNWFmZWM0MGE0NGE0OGIyYzYzOWEyMTYxMGVjOTAiLCJwIjoiaiJ9)

## Changes/solution

- Adds the inline policies(name and JSON formatted document)
- Adds permission boundary ARN and relationship to iamPolicy that matches that ARN

### CODE REVIEW

File: src/services/iamRole/format.ts [hunk 1]
```diff
@@ -24,7 +24,8 @@ export default ({
     RoleLastUsed,
     AssumeRolePolicyDocument: assumeRolePolicy = '',
     MaxSessionDuration: maxSessionDuration = 0,
-    Policies: inlinePolicies = [],
+    InlinePoliciesName: inlinePolicies = [],
+    InlinePoliciesDocuments: inlineFormattedPolicies,
```
Comment (tyler-dunkel) [inline src/services/iamRole/format.ts:28]: I would just make this one object. `inlinePolicies` which is a array of obj [{ id: ..., name: ..., content: formatIamJsonPolicy()}]`

File: src/services/iamPolicy/schema.graphql [hunk 2]
```diff
@@ -8,6 +8,7 @@ type awsIamPolicy implements awsBaseService @key(fields: "id") {
   iamRoles: [awsIamRole] @hasInverse(field: iamAttachedPolicies)
   iamGroups: [awsIamGroup] @hasInverse(field: iamAttachedPolicies)
   iamUsers: [awsIamUser] @hasInverse(field: iamAttachedPolicies)
+  permissionboundaryOf: [awsIamRole] @hasInverse(field: iamPermissionBoundaryPolicy)
```
Comment (tyler-dunkel) [inline src/services/iamPolicy/schema.graphql:11]: `permissionBoundaryOf`

File: src/services/iamRole/format.ts [hunk 3]
```diff
@@ -43,7 +43,12 @@ export default ({
     rawPolicy: assumeRolePolicy,
     assumeRolePolicy: formatIamJsonPolicy(assumeRolePolicy),
     maxSessionDuration,
-    inlinePolicies,
+    inlinePolicies: inlinePolicies.map(
+      ({ name: inlinePolicyName, document: inlinePolicyDocument }) => ({
+        name: inlinePolicyName,
```
Comment (tyler-dunkel) [inline src/services/iamRole/format.ts:48]: This needs a unique id does it not?

Comment (hjaraujof) [reply src/services/iamRole/format.ts:48]: Yes, it does. I'll fix it right away

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**